scripts/cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+  echo -e "${INF} cleaning up"
+
```

Comment:
```init.sh``` is ```source```d on top of ```mir-perf.sh```, so the content is automatically visible in all subsequent sourced files.